resources/assets/js/file-download.js

```diff
@@ -0,0 +1,18 @@
+window.fileDownload = () => {
+    return {
+        save: function (filename, data, dataType, extension) {
+            let blob = new Blob([data], { type: dataType });
+
+            if (window.navigator.msSaveOrOpenBlob) {
+                window.navigator.msSaveBlob(blob, `${filename}.${extension}`);
```

Comment:
probably best to remove this part as it's deprecated anyway: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/msSaveOrOpenBlob